ci-operator/step-registry/cucushift/chainupgrade/toimage/cucushift-chainupgrade-toimage-commands.sh

```diff
@@ -254,6 +254,61 @@ EOF
     ansible-playbook -i "${SHARED_DIR}/ansible-hosts" /tmp/repo.yaml -vvv
 }
 
+# Do sdn migration to ovn since sdn is not supported from 4.17 version
+function sdn2ovn(){
+    TARGET=${TARGET:-OVNKubernetes}
+    timeout 600s bash <<EOT
```

Comment:
if we run sdn2ovn after health_check, I think we don't need to wait co ready anymore.

Reply:
Agree. In the toimage step, we always run health check, also including the co ready testing.

Reply:
OK, I removed this steps. thanks.